docs/release_notes/mar2023.md

```diff
@@ -8,6 +8,49 @@ tags:
   - changelog
 ---
 
+## 2.10.0.0 (Mar 31, 2023)
+
+- Prophecy Python libs version: 1.4.8
+- Prophecy Scala libs version: 7.0.16
+
+### Updates {#UpdatesRelease2100}
+
+#### Support for Personal Access Tokens
+
+A Personal Access Token(PAT) is the token which customer can use for authentication to our API servers.
+This will enable customers to use PAT in their scripts and integrate Prophecy seamlessly to their data visualization ecosystem.
+They would be able to access Prophecy metadata through APIs and create a single pane of glass interface for their data stewards and engineers to see pipelines, datasets, Lineage, etc.
+Please see [here](./../metadata/personalAccessTokens) for more details on this feature.
+
+### Improvements {#ImprovementsRelease2100}
+
+#### Search box on available clusters
+
+To help Users find there clusters faster, we have added Search capability when listing down the clusters in Attach cluster screens. Please see below video for example.
+
+![SearchCluster](img/ClusterSearch.gif)
+
+#### Performance Improvements for Projects with large Number of Entities
+
+With this release, we have Optimized the Project Browser to handle Large Projects (Project With of Pipelines/Datasets in orders of 500s).
```

Comment:
With this release, we have optimized the Project Browser to handle Projects with more than 500 Pipelines/Datasets. User should not see delays and should be able to seamlessly browse Entities in large Projects.